src/Augmint.ts

```diff
@@ -14,18 +14,38 @@ import * as Errors from "./Errors";
 import { EthereumConnection, IOptions } from "./EthereumConnection";
 import { Exchange, IExchangeOptions } from "./Exchange";
 import * as gas from "./gas";
-import { ILoanManagerOptions, LoanManager } from "./LoanManager";
+import { Loan } from "./Loan";
+import { LoanManager } from "./LoanManager";
+import { LoanProduct } from "./LoanProduct";
 import { Rates } from "./Rates";
 import { Transaction } from "./Transaction";
+import { Tokens, Wei } from "./units";
 
 interface IDeployedEnvironmentStub {
     [propName: string]: IDeploymentItem[];
 }
 
+interface ILoanCount {
+    loanManagerAddress: string
+    loanCount: number
+}
+
+const loanManagers: Map<string, LoanManager> = new Map<string, LoanManager>();
```

Comment:
shouldn't this be in the `Augmint` class, not a global?

Reply:
yep, it should.

Reply:
done